### PR TITLE
Attempt to upgrade to Scala 2.13(.2)

### PR DESCRIPTION
**Pull request by viktorklang**
@jroper would this help you?

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -59,7 +59,7 @@ val AkkaHttpVersion = "10.1.12"
 val AkkaManagementVersion = "1.0.5"
 val AkkaPersistenceCassandraVersion = "0.102"
 val PrometheusClientVersion = "0.6.0"
-val ScalaTestVersion = "3.0.5"
+val ScalaTestVersion = "3.0.8"
```
Comment (viktorklang) [inline build.sbt:62]: This was the closest version cross-published for 2.13 that I found

File: java-support/src/main/scala/io/cloudstate/javasupport/impl/AnySupport.scala [hunk 2]
```diff
@@ -158,7 +158,8 @@ object AnySupport {
         if (results.contains(descriptorName)) results
         else {
           val withDesc = results.updated(descriptorName, descriptor)
-          flattenDescriptors(withDesc, descriptor.getDependencies.asScala ++ descriptor.getPublicDependencies.asScala)
+          flattenDescriptors(withDesc,
+                             descriptor.getDependencies.asScala.toSeq ++ descriptor.getPublicDependencies.asScala)
```
Comment (viktorklang) [inline java-support/src/main/scala/io/cloudstate/javasupport/impl/AnySupport.scala:162]: `asScala` no longer produces a Seq (it's a Buffer)

**Comment by viktorklang** [comment]
@pvlugter Merge this at your convenience.

**Comment by pvlugter** [comment]
Rebased on master, resolved conflicts, bumped to scala 2.13.3. Letting the tests run again before merge.

**Pull request merged**